---
bool success
gpd_ros/Grasp best_grasp
